config-development.json

```diff
@@ -0,0 +1,25 @@
+{
+    "system": {
+      "logging": "debug",
+      "baseurl": "openhab-cloud.org:443"
```

Comment:
what basurl is that? If it is meant as a placeholder, better mark it as such (i.e. `<your-openhab-cloud-host>:443`)